etc/devices/redfishpower-cray-ex-rabbit.dev

```diff
@@ -0,0 +1,160 @@
+# Support for Redfish Rest Interface
+#
+# How blades and switches in a HPE Cray Supercomputing EX Chassis
+# (abbreviated CrayEX) are populated has a large effect on
+# configuration.  Tweaks to this device file may be needed.  See the
+# section "UPDATING REDFISHPOWER DEVICE FILES" in redfishpower(8) for
+# additional tips.
```

Comment:
I would delete all the docs and point to the other dev script.

Just say "This is a variant of the cray-ex chassis populated with rabbit storage hardware.  Refer to cray-ex for info specific to that chassis." or similar.

Reply:
I think I need to keep the examples though ... right?  They show rabbits afterall.

Lemme trim it down based on your suggestion though.